pages/integrations/inputs/o365_input.rst

```diff
@@ -0,0 +1,54 @@
+.. _o365_input:
+
+***********************
+Office 365 Integrations
+***********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+               It is meant for the hosted "online" office and do not use this to collect logs from a windows server, in your data center.
+
+Prerequisites
+=============
+You have an application in Azure AD, a subscription to Office 365 and a subscription to Azure.
+
+Why does this plugin exist ?
+============================
+Office 365 Integrations plugin, collects data from Windows Logs, Email, File Sharing and Data Loss Prevention Logs in one input.
+This plugin retrieves data for each tenant, using continuous polling and stores the data(user activity) in `elastic <https://en.wikipedia.org/wiki/Elasticsearch>`_ to detect threats.
+
+How do I get this plugin running ?
+==================================
+you should be able to login into the following websites with your exempli@gratia.onmicrosoft.com account.
+
+ - `office 365 <https://www.office.com/?auth=2>`_
+ - `azure Portal <https://portal.azure.com/#home>`_
+ - `turn on <https://protection.office.com/unifiedauditlog>`_ unified logging on friday and perform a search on monday,to see results.
+
+once you login into Azure portal,you can obtain the essentials for running this plugin.Please collect items 1-3 from the azure portal.
+Leave the defaults for the rest of the fields while navigating the **graylog-o365-input-plugin** wizard.You must be up and running with this input **Office 365 Log Events**
+plugin in a few minutes.If the plugin stops **RUNNING** completely, due to any failures, click **Start input** to collect data from the last **CHECKPOINT_TIME** of failure.
+
+
+1) ``Directory (tenant) ID``
+2) ``Application (client) ID``
+3) ``Client Secret``
+4) ``Enterprise and GCC Government plans`` is the most common subscription type.
+
+This `input <http://localhost:8080/system/inputs>`_ plugin polls data internally using, https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-reference.
```

Comment:
why is the input link to a localhost address?

Reply:
will changing it to ipaddress help?

Reply:
Do we need a Hyperlink here at all?  Can we just say it without the link?